proto/metadata.proto

```diff
@@ -298,4 +298,16 @@ message Metadata {
   // The `platform` field for each `platform_exceptions` should be mutually
   // exclusive. Duplicate matches will result in a test failure.
   repeated PlatformExceptions platform_exceptions = 5;
+
+  enum Tags {
```

Comment:
nits/questions: 

Instead of `DEFAULT` should we rename this to `ALL` so that it reflects what the default is meant to do.

Also instead of tags, should this be renamed to `TestArea` to be more clear, because technically all fields in the metadata proto are like tags?

Reply:
proto format should be

TAGS_UNSPECIFIED
TAGS_AGGREGATION
TAGS_DATA_CENTER_EDGE ...

Reply:
Done